### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by ccobasi**
- Configure redux store
- Import combineReducers
- Create a book action

### CODE REVIEW

File: s/src/redux/categories/categories.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (arslanbisharat) [inline s/src/redux/categories/categories.js]: - [ ] Please don't disable eslint rules. There are here to help us write best code possible.

File: s/src/redux/configureStores.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (arslanbisharat) [inline s/src/redux/configureStores.js]: - [ ] Please don't disable eslint rules. There are here to help us write best code possible.

File: src/components/Add.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import { useDispatch } from 'react-redux';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (arslanbisharat) [inline src/components/Add.js]: - [ ] Please don't disable eslint rules. There are here to help us write best code possible.

File: src/redux/categories/categories.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (arslanbisharat) [inline src/redux/categories/categories.js]: - [ ] Please don't disable eslint rules. There are here to help us write best code possible.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] you have a directory S where you have added your project. Please move your project out.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Emmyn5600 — APPROVED** [review]
Hi, 

Good job so far! 👏🏼 @ccobasi 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.
 
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**